src/Maestro/Microsoft.DotNet.Maestro.Tasks.Tests/SharedObjects.cs

```diff
@@ -0,0 +1,627 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using FluentAssertions;
+using Microsoft.DotNet.Maestro.Client.Models;
+
+namespace Microsoft.DotNet.Maestro.Tasks.Tests
+{
+    public class SharedObjects
+    {
+        public const string Commit = "e7a79ce64f0703c231e6da88b5279dd0bf681b3d";
+        public const string AzureDevOpsAccount1 = "dnceng";
+        public const int AzureDevOpsBuildDefinitionId1 = 6;
+        public const int AzureDevOpsBuildId1 = 856354;
+        public const string AzureDevOpsBranch1 = "refs/heads/master";
+        public const string AzureDevOpsBuildNumber1 = "20201016.5";
+        public const string AzureDevOpsProject1 = "internal";
+        public const string AzureDevOpsRepository1 = "https://dnceng@dev.azure.com/dnceng/internal/_git/dotnet-arcade";
+        public const string LocationString = "https://dev.azure.com/dnceng/internal/_apis/build/builds/856354/artifacts";
+
+        #region SigningInformation
+        public static readonly List<SigningInformation> ExpectedSigningInfo = new List<SigningInformation>()
```

Comment:
Oh my gosh... This file is _really_ big, and a lot of this data is specific to specific tests.  I would put objects that particular tests/fixtures need inside those tests, _even if that means duplicating some of it for different fixtures_.  Clarity of a test/failure is really important for test longevity, and that's usually improved by getting the test data as close to the test as possible.

Reply:
Done